### ISSUE TITLE
Failure at sentry:validate_config

### ISSUE THREAD

**Issue by iwaseasahi** [codeur/capistrano-sentry#8]
## gem version
v0.4.1

## error message

```
  INFO [sentry:validate_config] Validating Sentry notification config
(Backtrace restricted to imported tasks)
cap aborted!
NoMethodError: undefined method `blank?' for #<String:0x00005577f98d6c58>

Tasks: TOP => deploy:starting => sentry:validate_config
(See full trace by running task with --trace)
The deploy has failed with an error: undefined method `blank?' for #<String:0x00005577f98d6c58>
```

## code location

https://github.com/codeur/capistrano-sentry/blob/b9a12b221ad18dba36077f7ce6566067a44b0e2d/lib/capistrano/tasks/sentry.rake#L23

## causing
`blank?` is the method of Ruby on Rails.

https://api.rubyonrails.org/v6.0.3.1/classes/String.html#method-i-blank-3F

It is not in method of ruby's string class.

```
irb(main):003:0> 'string'.respond_to?(:blank?)
=> false
```

## comment
I'm at work so I created an issue first.
When I find the time, I will create a pull request.

Thanks for your gem.

**Comment by burisu** [comment]
Thank you for your help. I "rubocoped" the code too quickly. I'll fix that now.

**Comment by burisu** [comment]
A new gem has been released (v0.4.2)
